thunder/executors/nvfuserex_impl.py

```diff
@@ -2085,6 +2087,7 @@ def copy_(
     *,
     fd: FusionDefinition,
     lc_to_nv_map: dict,
+    grad_enabled: bool,
```

Comment:
What is the behaviour for `nvfuser`? I think that we ignore this argument. Should we raise a warning instead?

Reply:
Yes, the argument is ignored, and nvfuser does not fail.